Troubleshooting.md

```diff
@@ -393,3 +396,11 @@ For backward compatibility it is possible to restore Gson's old behavior of allo
 
 - This does not solve any of the type-safety problems mentioned above; in the long term you should prefer one of the other solutions listed above. This system property might be removed in future Gson versions.
 - You should only ever set the property to `"true"`, but never to any other value or manually clear it. Otherwise this might counteract any libraries you are using which might have deliberately set the system property because they rely on its behavior.
+
+## R8 / ProGuard
+
+If you use Gson as a dependency in an Android project which uses R8 as a default compiler you don’t have to do anything.
```

Comment:
Minor: Maybe use a regular `'` for consistency with the remainder of this document.
```suggestion
If you use Gson as a dependency in an Android project which uses R8 as a default compiler you don't have to do anything.
```